CHANGES.rst

```diff
@@ -1,6 +1,7 @@
 v39.2.0
 -------
 
+* #97: PEP 420: introduce find_packages_ns()
```

Comment:
For the future, changes go in their own separate files in the `changelog.d` folder (for various reasons, including avoiding merge conflicts with everyone editing the same CHANGELOG file). In this case you should create a stub called `changelog.d/1312.change.rst`.